recipes/treematching/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "treematching" %}
+{% set version = "0.1.20210513" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/LionelAuroux/treematching
+  git_tag: 5ebc2474d8720d87569cf414bd0275010edf8f19
+
+build:
+  number: 0
```

This can be `noarch: python`
```suggestion
  number: 0
  noarch: python
```